test/ExecutionFramework.h

```diff
@@ -293,6 +295,12 @@ class ExecutionFramework
 	u256 m_gasUsed;
 };
 
+#define ABI_CHECK(RESULT, EXPECTATION) do { \
```

Comment:
Can you actually go lowercase since that is what we have in the rest of the code?

Reply:
Hm, also for preprocessor macros?

Reply:
Hm, mixed, but I'll make it lowercase.

Reply:
That is what we have. I don't think I've seen anywhere uppercase for macros, but could make sense, though I'd prefer to do a single sweeping change to update all macros for that style if decided on.